api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -284,6 +284,7 @@ private WorkspaceResource buildRecentResource(
 
   private void buildFromDbWorkspace(WorkspaceResource resource, DbWorkspace dbWorkspace) {
     resource.setCdrVersionId(commonMappers.cdrVersionToId(dbWorkspace.getCdrVersion()));
+    resource.setAccessTierShortName(dbWorkspace.getCdrVersion().getAccessTier().getShortName());
```

Comment:
just above this: // TODO RW-5638 reimplement all this in MapStruct

hard agree